rfc-000-external-route.md

```diff
@@ -0,0 +1,45 @@
+# External routes
+
+## Summary
+
+Create an `external_route` document type to represent pages that aren't part
+of GOV.UK but are relavent to our users.
+
+## Problem
+
+Not everything government related is on GOV.UK.
+
+Search admin stores "External links", which can be returned for certain search terms.
+
+For example:
+- searches for "mp" can return the parliament.uk page http://www.parliament.uk/mps-lords-and-offices/mps/
+- searches for "fit for work" can return http://fitforwork.org/
+- searches for council names can return the council website
+
+We are currently changing search indexing to source all its content from the publishing API, but external links
+have never been part of the publishing API.
+
+## Proposal
+
+Create an `external_route` document type so we can store these links in the publishing API.
+
+Create an `external_route` schema.
+
+The schema MUST store in the details hash:
+- `hidden_search_terms` - a set of search keywords/phrases that should route users to the page. [This field has already been added for smart answers](https://github.com/alphagov/govuk-content-schemas/pull/685/files).
+-  `url` - a URL for the external resource
+- `comment` - an internal description of why the link is relevant to GOV.UK users
```

Comment:
Is this the same as the field in search-admin? Maybe we don't want to display this publicly.

Reply:
Yes it is. It's fine to take this out, but I don't think there's a strong reason it can't be public. We would have to clean up the values a bit first though.

Reply:
I've updated the RFC to take this out as we don't need it.